model/driver/src/icon4py/model/driver/icon4py_driver.py

```diff
@@ -351,7 +351,7 @@ def initialize(
     log.info("initializing diffusion")
     diffusion_params = diffusion.DiffusionParams(config.diffusion_config)
     exchange = decomposition.create_exchange(props, decomp_info)
-    diffusion_granule = diffusion.Diffusion(exchange)
+    diffusion_granule = diffusion.Diffusion(exchange=exchange, backend=None)
```

Comment:
This will run on field view embedded backend. Can you pass here the gtfn_cpu for now?